stubs/protobuf/google/protobuf/compiler/plugin_pb2.pyi

```diff
@@ -101,6 +104,12 @@ class CodeGeneratorRequest(google.protobuf.message.Message):
         fully qualified.
         """
     @property
+    def source_file_descriptors(self) -> google.protobuf.internal.containers.RepeatedCompositeFieldContainer[google.protobuf.descriptor_pb2.FileDescriptorProto]:
```

Comment:
This code could solve the problem I pointed out in https://github.com/python/typeshed/issues/11685 . (We can close https://github.com/python/typeshed/issues/11685 .)